oauth/client.go

```diff
@@ -0,0 +1,84 @@
+// Package oauth provides the OAuth APIs
+package oauth
+
+import (
+	"fmt"
+	stripe "github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go/form"
+	"net/http"
```

Comment:
And same comment as in `client_test.go` about reorganizing imports into two groups.